libs/database/Cargo.toml

```diff
@@ -21,12 +21,12 @@ serde_json.workspace = true
 tonic-proto.workspace = true
 
 sqlx = { workspace = true, default-features = false, features = [
-  "postgres",
-  "chrono",
-  "uuid",
-  "macros",
-  "runtime-tokio-rustls",
-  "rust_decimal",
+    "postgres",
+    "chrono",
+    "uuid",
+    "macros",
+    "runtime-tokio-rustls",
+    "rust_decimal",
 ] }
```

Comment:
@appflowy @khorshuheng I remember seeing PR having indentation spaces change from 2 to 4, and now back to 2. we should keep it consistent.

Reply:
I've set the tab_spaces =2 In rustfmt.toml. But i am not sure the cargo.toml will use the rustfmt.toml configuration

Reply:
It was due to editor auto formatting. We can choose either 2 or 4, i can adjust my editor accordingly.

The main problem is that cargo fmt doesn't support formatting Cargo.toml, unfortunately, though there have been attempts to introduce this.

Reply:
it seems like my IDE auto format the cargo.toml file. I will try set tab_spaces = 2. Let's keep 2

Reply:
I found it.

<img width="1109" alt="image" src="https://github.com/user-attachments/assets/f7166d7d-a1bf-4ae4-b9d8-3687e98efed0">